NightscoutUploadKit/NightscoutUploader.swift

```diff
@@ -169,7 +169,7 @@ public class NightscoutUploader: NSObject {
                 "timestamp": pumpDateStr,
                 "bolusiob": status.iob,
             ],
-            "reservoir": status.reservoirRemainingUnits,
+            "reservoir": Double(round(status.reservoirRemainingUnits*1000)/1000,
```

Comment:
That last value was an example from bolusiob, which I just noticed has the same issue.   Here is an example value from the reservoir field: 42.90000000000001